Seems a header file could not be found:
```
/Users/vagrant/azure-agent/_work/2/s/buildtrees/ogre/src/c710e36bc0-fd78f2aff5/OgreMain/include/OgrePlatform.h:150:13: fatal error: 'Availability.h' file not found
#   include "Availability.h"
            ^~~~~~~~~~~~~~~~
1 error generated.
```

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -1172,6 +1172,7 @@ ogdf:x64-uwp            = skip
 ogdf:x64-windows        = skip
 ogdf:x64-windows-static = skip
 ogdf:x86-windows        = skip
+ogre:x64-osx=fail
 # Conflicts with ogre
 ogre-next:arm64-windows      = skip
 ogre-next:arm-uwp            = skip

